packages/admin-ui/src/table/styles/table-cell.styles.ts

```diff
@@ -6,6 +6,8 @@ export const baseline = style({
   display: 'table-cell',
   verticalAlign: 'middle',
   paddingX: '$xl',
+  size: 44,
+  minHeight: 44,
```

Comment:
Pixel police passing through 🚓

Reply:
Maybe it's in pixels to help with dynamic size calculations 🤔